README.md

```diff
@@ -99,51 +103,6 @@ if no other config file is specified using the command line option `-c/--config-
 For more information check [usage](doc/usage.md).
 
 
-## Clients
-
-In the [clients](./clients) folder, you can find differents clients for kuksa-val-server
-
-### Python test client
-
-You can use the [testclient](./clients/vss-testclient/) to test basic functions like _getMetaData_, _getValue_ and _setValue_ requests.
-
-After starting the kuksa server using `./kuksa-val-server` command. For testing you can downlaod and load the the testclient docker image similarly to the main server and start it like this
-
-```bash
-docker run -it --rm --net=host amd64/kuksa-vssclient:b3084b9
-```
-
-or you can start the testclient as follows:
-
-```bash
-cd clients/vss-testclient
-pipenv sync
-pipenv run python testclient.py
-```
-
-
-Check the [testclient readme](./clients/vss-testclient/README.md) for more detailled installaton instruction.
-
-Once the testclient is up, you can start exploring the VSS structure and datapoints:
-
-![Alt text](./doc/pictures/testclient_basic.gif "test client usage")
-
-Using the testclient, it is also possible to update and extend the VSS data structure. More details can be found [here](./doc/liveUpdateVSSTree.md).
-
-
-### Feeder
-
-We provider currently the following feeders:
-- [DBC feeder](./clients/feeder/dbc2val)
```

Comment:
Links broken

Reply:
this part is removed

Reply:
Sorry wrong location. Here: https://github.com/boschresearch/kuksa.val/tree/111_python_package/kuksa_feeders